cdtb/storage.py

```diff
@@ -476,3 +479,9 @@ def parse_storage_component(storage: Storage, component: str) -> Union[None, Pat
         return patch
     else:
         return storage.patch_element(name, version, stored=version is not None)
+
+# Import known storages to register them
+# Put the imports at the end to avoid circular dependenceis
+from . import rads as _rads  # noqa
+from . import patcher as _patcher   # noqa
```

Comment:
What does this do exactly?

Reply:
It imports the storage subclasses so they are registered in `_storage_registry` when the are created (because of the `StorageRegister` metaclass).

Reply:
I see, that's a bit of an awkward logic. Looks like `__init__.py` previously did this implicit registering. Fine I guess.